src/dispatch/static/dispatch/src/tag/TagPicker.vue

```diff
@@ -269,14 +283,28 @@ const emit = defineEmits(["update:modelValue"])
 
 function validateTags(value) {
   const project_id = props.project?.id || 0
-  const all_tags_in_project = value.every((tag) => tag.project?.id == project_id)
+  var all_tags_in_project = false
+  if (project_id) {
+    all_tags_in_project = value.every((tag) => tag.project?.id == project_id)
+  } else {
+    const project_name = props.project?.name
+    if (!project_name) {
+      error.value = true
+      dummyText.value += " "
+      return
+    }
+    all_tags_in_project = value.every((tag) => tag.project?.name == project_name)
+  }
   if (all_tags_in_project) {
-    error.value = true
-    dummyText.value += " "
+    if (are_required_tags_selected(value)) {
+      error.value = true
+    } else {
+      error.value = "Please select at least one tag from each required category"
```

Comment:
We can educate the user how to address the error here. Correct me if I am wrong, but right now it's not immediately obvious "which" categories are required.

Reply:
I added a screenshot to the description showing how the Required label shows up in the selector. Let me know if you think of a better way to call this out.

Reply:
I think we could include "The following categories are required: example, test, pizza" to the main error message.

Reply:
Good idea! I've added a parenthetical list of required type(s) to the error message.